Content.Server/Motd/MOTDSystem.cs

```diff
@@ -0,0 +1,102 @@
+using Content.Server.Chat.Managers;
+using Content.Server.GameTicking;
+using Content.Shared.CCVar;
+using Content.Shared.Chat;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Configuration;
+
+namespace Content.Server.Motd;
+
+/// <summary>
+/// The system that handles broadcasting the Message Of The Day to players when they join the lobby/the MOTD changes/they ask for it to be printed.
+/// </summary>
+public sealed class MOTDSystem : EntitySystem
+{
+    [Dependency] private readonly IChatManager _chatManager = default!;
+    [Dependency] private readonly IConfigurationManager _configurationManager = default!;
+
+    /// <summary>
+    /// The cached value of the Message of the Day. Used for fast access.
+    /// </summary>
+    private string _messageOfTheDay = "";
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        _messageOfTheDay = _configurationManager.GetCVar(CCVars.MOTD);
+        _configurationManager.OnValueChanged(CCVars.MOTD, OnMOTDChanged);
```

Comment:
MOTD has an arg to invoke immediately.

Reply:
Right. I'll use that. I suppose broadcasting to players won't have any effect on init because the players haven't connected yet?

Reply:
Done.